file properties on context menu

aditya-mitra opened the pull request:
## Summary

## Subtasks Checklist

## Breaking Changes

## References
closes part of https://tsu.atlassian.net/browse/IR-2624

[Figma Design](https://www.figma.com/design/DXwFHGa5g0peqqSjyjpxQz/Studio-Design?node-id=4452-11971&t=HMvscZdomTo77dwk-0)

## QA Steps

![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/605ea463-7682-4213-b71a-ccb24d1c8275)

aditya-mitra committed:
file properties on context menu

aditya-mitra committed:
show extra fields for resource properties only

aditya-mitra committed:
Merge branch 'dev' into ir-2624

aditya-mitra committed:
fix check errors

CITIZENDOT committed:
Merge branch 'dev' into ir-2624

CITIZENDOT reviewed (changes requested):
- triangle count, texture count and Author properties are missing (compared to design). is this intentional?
![image](https://github.com/EtherealEngine/etherealengine/assets/52322531/81363883-a71b-4de2-8bfa-0981517b2402)
- fontFamily is `Figtree` in design
- Just a minor UX issue (not much important): 
![image](https://github.com/EtherealEngine/etherealengine/assets/52322531/165d1fd3-f880-43ed-8b79-b18791c82c9d)
When clicked `Enter` after typing a tag, it would be nice if that gets added. (just like clicking `+` button)

aditya-mitra committed:
added the author field

aditya-mitra committed:
avoid duplicate and empty tag addition

aditya-mitra committed:
Merge branch 'dev' into ir-2624

aditya-mitra committed:
remove unwanted lines

aditya-mitra commented:
> triangle count, texture count and Author properties are missing (compared to design). is this intentional?

The texture and triangle count need to be present in the model (static-resource). Currently we do not get this property from the API. 
Added the author property.
![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/e2a14a65-9d3b-414f-babc-aa4a401f76be)

> fontFamily is Figtree in design

added

> Just a minor UX issue (not much important):

added

HexaField commented:
FYI, we do have a field for triangle and texture count, the `stats` field, but we have not populated these yet. 

See https://github.com/etherealengine/etherealengine/blob/dev/packages/server-core/src/media/static-resource/static-resource-helper.ts#L191
